src/garage/experiment/deterministic.py

```diff
@@ -2,26 +2,36 @@
 import random
 import sys
 
+from dowel import logger
 import numpy as np
 
 seed_ = None
 
 
-def set_seed(seed):
+def set_seed(seed, pytorch=False):
```

Comment:
i'm not in favor of this option. if you want your experiment to be deterministic, you set the seed.

Reply:
I added this because making PyTorch deterministic has performance impact, but on second thought, it doesn't really make sense since users won't be running both PyTorch and Tensorflow experiments together. I will remove this.

Reply:
you can print a warning with `warnings.warn` that this has a negative impact on performance.

BTW, i thought it only impacts performance on GPU, right?

Reply:
yes, you are right. I changed the text.